Library/Include/Babylon/RuntimeAndroid.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include "Runtime.h"
+
+namespace babylon
+{
+    class RuntimeAndroid final : public Runtime
+    {
+    public:
+        explicit RuntimeAndroid(void* nativeWindowPtr);
```

Comment:
void* should probably be ANativeWindow*?

Reply:
Fixed